ubuntu2004/profiles/cis_level1_server.profile

```diff
@@ -0,0 +1,653 @@
+
+    ### 1.1.4 Ensure nosuid option set on /tmp partition (Automated)
+    - mount_option_tmp_nosuid
+
+    ### 1.1.5 Ensure noexec option set on /tmp partition (Automated)
+    - mount_option_tmp_noexec
+
+    ### 1.1.6 Ensure /dev/shm is configured (Automated)
+    # Skip due to being handled by systemd and ensured by follow-on tests
+
+    ### 1.1.7 Ensure nodev option set on /dev/shm partition (Automated)
+    - mount_option_dev_shm_nodev
+
+    ### 1.1.8 Ensure nosuid option set on /dev/shm partition (Automated)
+    - mount_option_dev_shm_nosuid
+
+    ### 1.1.9 Ensure noexec option set on /dev/shm partition (Automated)
+    - mount_option_dev_shm_noexec
+
+    ### 1.1.10 Ensure separate partition exists for /var (Automated)
+    # Skip due to being Level 2
+
+    ### 1.1.11 Ensure separate partition exists for /var/tmp (Automated)
+    # This is a Level 2 test but is needed for 1.1.12-14
+    - partition_for_var_tmp
+
+    ### 1.1.12 Ensure nodev option set on /var/tmp partition (Automated)
+    - mount_option_var_tmp_nodev
+
+    ### 1.1.13 Ensure nosuid option set on /var/tmp partition (Automated)
+    - mount_option_var_tmp_nosuid
+
+    ### 1.1.14 Ensure noexec option set on /var/tmp partition (Automated)
+    - mount_option_var_tmp_noexec
+
+    ### 1.1.15 Ensure separate partition exists for /var/log (Automated)
+    # Skip due to being Level 2
+
+    ### 1.1.16 Ensure separate partition exists for /var/log/audit (Automated)
+    # Skip due to being Level 2
+
+    ### 1.1.17 Ensure separate partition exists for /home (Automated)
+    # This is a Level 2 test but is needed for 1.1.18
+    - partition_for_home
+
+    ### 1.1.18 Ensure nodev option set on /home partition (Automated)
+    - mount_option_home_nodev
+
+    ### 1.1.19 Ensure nodev option set on removable media partitions (Manual)
+    # Skip due to being a manual test
+
+    ### 1.1.20 Ensure nosuid option set on removable media partitions (Manual)
+    # Skip due to being a manual test
+
+    ### 1.1.21 Ensure noexec option set on removable media partitions (Manual)
+    # Skip due to being a manual test
+
+    ### 1.1.22 Ensure sticky bit is set on all world-writable directories (Automated)
+    - dir_perms_world_writable_sticky_bits
+
+    ### 1.1.23 Disable Automounting (Automated)
+    - service_autofs_disabled
+
+    ### 1.1.24 Disable USB Storage (Automated)
+    - kernel_module_usb-storage_disabled
+
+    ## 1.2 Configure Software Updates ##
+    ### 1.2.1 Ensure package manager repositories are configured (Manual)
+    # Skip due to being a manual test
+
+    ### 1.2.2 Ensure GPG keys are configured (Manual)
+    # Skip due to being a manual test
+
+    ## 1.3 Configure sudo ##
+    ### 1.3.1 Ensure sudo is installed (Automated)
+
+    ### 1.3.2 Ensure sudo commands use pty (Automated)
+
+    ### 1.3.3 Ensure sudo log file exists (Automated)
+
+    ## 1.4 Filesystem Integrity Checking ##
+    ### 1.4.1 Ensure AIDE is installed (Automated)
```

Comment:
Just in case you're interested, I think [`package_aide_installed`](https://github.com/ComplianceAsCode/content/tree/master/linux_os/guide/system/software/integrity/software-integrity/aide/package_aide_installed) together with [`aide_build_database`](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/software/integrity/software-integrity/aide/aide_build_database/rule.yml) would work here.

Though, perhaps `aide_build_database` should be updated to what's in the current guidance:

> ```console
> # aideinit
> # mv /var/lib/aide/aide.db.new /var/lib/aide/aide.db
> ```